roles/common/files/uug_ansible_wrapper.py

```diff
@@ -37,6 +37,7 @@
     'CS 261': 'cs261',
     'CS 354': 'cs354',
     'CS 361': 'cs361',
+    'CS 480': 'cs480'
```

Comment:
```suggestion
    'CS 430': 'cs430'
```

Reply:
I feel like my review was pointless, having missed this. :)